libpod/rootless_cni_linux.go

```diff
@@ -233,14 +225,13 @@ func ensureRootlessCNIInfraContainerRunning(ctx context.Context, r *Runtime) (*C
 }
 
 func startRootlessCNIInfraContainer(ctx context.Context, r *Runtime) (*Container, error) {
-	imageName, ok := rootlessCNIInfraImage[runtime.GOARCH]
-	if !ok {
-		return nil, errors.Errorf("cannot find rootless-podman-network-sandbox image for %s", runtime.GOARCH)
-	}
+	imageName := "rootless-cni-infra"
 	logrus.Debugf("rootless CNI: ensuring image %q to exist", imageName)
-	newImage, err := r.ImageRuntime().New(ctx, imageName, "", "", nil, nil,
-		image.SigningOptions{}, nil, util.PullImageMissing)
+	newImage, err := r.ImageRuntime().NewFromLocal(imageName)
 	if err != nil {
+		if errors.Cause(err) == define.ErrNoSuchImage {
+			return nil, errors.Errorf("rootless CNI infra image not present - please build image from https://github.com/containers/podman/blob/v2.2.1/contrib/rootless-cni-infra/Containerfile and tag as %q", imageName)
```

Comment:
I know it's long already, but could we give them the command to use to build the image instead of just the Containerfile?

Reply:
I don't think so - the actual build process is getting very complicated because we can't just do a `podman build $URL` (you need to pull down not just the containerfile, but the directory containing it, and then do a `podman build` on that)

Reply:
We have a separate card to document this elsewhere, which is where I think that sort of documentation will need to land.

Reply:
What about letting Podman itself execute `git clone ...  && git checkout ... && podman build -t rootless-cni-infra .`

Reply:
Problem is disconnected environments - we can't guarantee the system has a working connection to the internet.